modules/exploits/openbsd/local/xorg_x11_suid_server.rb

```diff
@@ -0,0 +1,171 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::Common
+  include Msf::Post::File
+  include Msf::Exploit::FileDropper
+  include Msf::Post::Linux::Priv
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Xorg X11 Server SUID privilege escalation',
+      'Description'    => %q{
+        This module attempts to gain root privileges with SUID Xorg X11 server
+        versions prior to 1.20.3.
+
+        A permission check flaw exists for -modulepath and -logfile options when
+        starting Xorg prior to 1.20.3.  This allows unprivileged users with the
+        ability to log in to the system via console to escalate their privileges
+        and run arbitrary code under root privileges (CVE-2018-14665).
+
+        This module writes a cron job using the Xorg -logfile option. The job
+        will run a small script to launch an executable. It has been tested with
+        OpenBSD 6.3,6.4 and CentOS 7.  Xorg must have SUID permissions. Success
+        on CentOS depends on the session having console for starting Xorg
+        along with selinux settings, may work but is currently not supported.
+
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Narendra Shinde', # Discovery and exploit
+          'Raptor - 0xdea',  # Modified exploit for cron
+          'Aaron Ringo'      # Metasploit module
+        ],
+
+      'DisclosureDate' => 'Oct 25 2018',
+      'Platform'       =>  %w(linux unix openbsd),
```

Comment:
You should put this module in `multi` if you're going to keep these platforms.

Reply:
Also, your whitespace appears to be off for these few lines.

Reply:
Changed to just bsd for the current.  My whitespace always feels like it is off.  Do you guys have some kinnd of linter thing you recommend ?

Reply:
[RuboCop](https://github.com/rubocop-hq/rubocop) may be able to help you with style, but my editor (Vim) does the heavy lifting with automatic indentation.